voc/automatic_updates.go

```diff
@@ -27,9 +29,12 @@ package voc
 
 import "time"
 
+// AutomaticUpdates This feature is, e.g., available on some VM services to automatically update their software. It ensures that a resource is protected from tampering with its state.
 type AutomaticUpdates struct {
 	*Integrity
-	Enabled      bool          `json:"enabled"`
-	Interval     time.Duration `json:"interval"`
-	SecurityOnly bool          `json:"securityOnly"`
+	Enabled	bool	`json:"enabled"`
+	SecurityOnly	bool	`json:"securityOnly"`
+	// The interval refers to the update interval in days.
```

Comment:
woah nice! How did you do that now?

Reply:
That was a cramp and not easy at all at this confused format. 

How I did that? I have absolutely no idea. But now it works :)